.github/workflows/build.yaml

```
@@ -30,6 +30,12 @@ jobs:
       - name: Pull docker.io/library/nginx:1.21.6
         run: docker pull docker.io/library/nginx:1.21.6
 
+      - name: Generate trivy vuln report for nginx image
+        run: |
+          cd ${{ github.workspace }}/test
+          mkdir data
+          trivy image --vuln-type os --ignore-unfixed -f json -o data/nginx.1.21.6.json docker.io/library/nginx:1.21.6
```

```suggestion
          trivy image --vuln-type os --ignore-unfixed -f json -o /tmp/nginx.1.21.6.json docker.io/library/nginx:1.21.6
```

you can remove the 2 lines above

change references to `${{ github.workspace }}/test/data` below too